.pre-commit-config.yaml

```diff
@@ -17,3 +17,10 @@ repos:
   hooks:
   - id: yamllint
     args: [-c=.yamllint]
+
+- repo: https://github.com/thoth-station/adviser.git
+  rev: 0669b0ea595e9eb59998efb11a236d3039dacdf8
+  hooks:
+    - id: validate-prescriptions
+      files: prescriptions/
+      exclude: prescriptions/_prescription_metadata.yaml
```

Comment:
yamllint is complaining: ` error    wrong indentation: expected 2 but found 4  (indentation) `